lib/python-beta/src/bailo/helper/schema.py

```diff
@@ -0,0 +1,62 @@
+from __future__ import annotations
+
+from typing import Any
+
+from bailo.core import Client, SchemaKind
+
+
+class Schema:
+    def __init__(
+        self,
+        client: Client,
+        schema_id: str,
+        name: str,
+        kind: SchemaKind,
+        json_schema: dict[str, Any],
+    ):
+        self.client = client
+        self.schema_id = schema_id
+        self.name = name
+        self.kind = kind
+        self.json_schema = json_schema
+
+    @classmethod
+    def from_id(cls, client: Client, schema_id: str):
+        res = client.get_schema(schema_id=schema_id)
+        schema_id, name, kind, json_schema = Schema.__unpack__(res)
+        schema = cls(
+            client=client,
+            schema_id=schema_id,
+            name=name,
+            kind=kind,
+            json_schema=json_schema,
+        )
+
+        return schema
+
+    @staticmethod
+    def __unpack__(res):
+        res = res['schema']
+        schema_id = res['id']
+        name = res['name']
+        kind = res['kind']
+        json_schema = res['jsonSchema']
+
+        if kind == 'model':
+            kind = SchemaKind.Model
+        if kind == 'accessRequest':
+            kind = SchemaKind.AccessRequest
+
+        return schema_id, name, kind, json_schema
```

Comment:
The object deconstruction is looking good. However I don't think think this a good use case for a `staticmethod` as the functionality is very object specific. I'd consider modifying your `classmethod` to use something `cls.__unpack()` after instantiating an object?

Reply:
Being a bit picky here but two prefixes and suffix underscores in the method name usually denotes magic methods. Private methods just have two prefixed underscores.